### DISCUSSION TITLE
CoreWCF (basichttp) in AWS Lambda

### DISCUSSION THREAD

**Discussion by DerekBeattieCG** [aws/aws-lambda-dotnet#1190] [Q&A] [answered]
Has anyone tried this or know if its possible?  I've pulled the source for aws-lambda-dotnet and CoreWCF and created a solution to try and work through it.  Both aws-lambda-dotnet and CoreWCF implement IServer so I'm trying to understand how to bridge the gap.  If anyone has any insight it would be appriciated.

**Comment by DerekBeattieCG** [comment, marked as answer]
This works as is, I'll attach a link to an example.

**Reply by closesimple-ms** [reply]
Hey Derek, do you have some stub code of this CoreWCF running on a lambda? I'm digging into the same idea.
